### ISSUE TITLE
Nightly log bundles contain current and previous nightly logs

### ISSUE THREAD

**Issue by cgtexmex** [vmware/vic#6095]
The nightly tests are creating log bundles with the current nightly test results and previous nightly results.  We should only be collecting the current results.  Example:

```
pwd
functional_logs_08_20_18_30_vic_13264/65/5-2-Cluster
5-2-Cluster cgreenwood$ ls
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-1018-container-logs.zip	VCH-0-2970-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-1079-container-logs.zip	VCH-0-3220-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-1590-container-logs.zip	VCH-0-3220-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-2949-container-logs.zip	VCH-0-5019-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-2965-container-logs.zip	VCH-0-5019-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-2970-container-logs.zip	VCH-0-5300-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-3220-container-logs.zip	VCH-0-5300-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-5019-container-logs.zip	VCH-0-5484-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-5300-container-logs.zip	VCH-0-5484-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-5484-container-logs.zip	VCH-0-5670-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-5670-container-logs.zip	VCH-0-5670-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-6290-container-logs.zip	VCH-0-6290-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-7315-container-logs.zip	VCH-0-6290-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-7434-container-logs.zip	VCH-0-7315-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-7859-container-logs.zip	VCH-0-7315-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-8493-container-logs.zip	VCH-0-7434-certs.zip
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-9418-container-logs.zip	VCH-0-7434-vmware.log
Manual-Test-Cases.Group5-Functional-Tests.5-2-Cluster-VCH-0-9520-container-logs.zip	VCH-0-7859-certs.zip
VCH-0-1018-certs.zip									VCH-0-7859-vmware.log
VCH-0-1018-vmware.log									VCH-0-8493-certs.zip
VCH-0-1079-certs.zip									VCH-0-8493-vmware.log
VCH-0-1079-vmware.log									VCH-0-9418-certs.zip
VCH-0-1590-certs.zip									VCH-0-9418-vmware.log
VCH-0-1590-vmware.log									VCH-0-9520-certs.zip
VCH-0-2949-certs.zip									VCH-0-9520-vmware.log
VCH-0-2949-vmware.log									log.html
VCH-0-2965-certs.zip									output.xml
VCH-0-2965-vmware.log									report.html
VCH-0-2970-certs.zip									vic-machine.log
```

**Comment by cgtexmex** [comment]
@mhagen-vmware why'd you remove the label `nightly`

**Comment by anchal-agrawal** [comment]
Fixed by @rajanashok in #6112.
